### PR TITLE
Updated audit logging related IGs.

### PR DESCRIPTION
**Pull request by ilyakubanov**
Ticket: https://spryker.atlassian.net/browse/CC-33753

### CODE REVIEW

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-feature.md [hunk 1]
```diff
@@ -0,0 +1,1366 @@
...
+```
+
+### 2) Set up configuration
+
+Add the following configuration to your project:
+
+| CONFIGURATION                                               | SPECIFICATION                                                                                                  | NAMESPACE                   |
+|-------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|-----------------------------|
+| SessionFileConstants::YVES_SESSION_FILE_PATH                | Defines the filesystem path for storing Yves sessions.                                                         | Spryker\Shared\SessionFile  |
+| SessionRedisConstants::YVES_SESSION_REDIS_PROTOCOL          | Defines the protocol used while connecting to Redis as Yves session storage.                                   | Spryker\Shared\SessionRedis |
```
Comment (AsonUnique) [inline _includes/pbc/all/install-features/202407.0/install-the-spryker-core-feature.md:978]: Deprecated

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-feature.md [hunk 2]
```diff
@@ -0,0 +1,1366 @@
...
+
+use Spryker\Glue\Log\Plugin\Log\GlueBackendSecurityAuditLoggerConfigPlugin;
+use Spryker\Glue\Log\Plugin\Log\GlueSecurityAuditLoggerConfigPlugin;
+use Spryker\Shared\Log\LogConstants;
+use Spryker\Zed\Log\Communication\Plugin\Log\MerchantPortalSecurityAuditLoggerConfigPlugin;
+use Spryker\Zed\Log\Communication\Plugin\Log\ZedSecurityAuditLoggerConfigPlugin;
+
+$config[PropelConstants::LOG_FILE_PATH_ZED]
+    = $config[LogConstants::LOG_FILE_PATH_GLUE]
+    =  'php://stdout';
```
[the remark this answers is not in the document]
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/install-the-spryker-core-feature.md:351]: The prefered value should be stdout

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-feature.md [hunk 3]
```diff
@@ -0,0 +1,1366 @@
...
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\Log\LogConstants;
+use Spryker\Yves\Log\Plugin\Log\YvesSecurityAuditLoggerConfigPlugin;
+
+$config[PropelConstants::LOG_FILE_PATH_ZED] = 'php://stdout';
```
[the remark this answers is not in the document]
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/install-the-spryker-core-feature.md:1139]: The preferred value should be stdout, fixed the key

File: _includes/pbc/all/install-features/202407.0/install-the-customer-account-management-feature.md [hunk 4]
```diff
@@ -0,0 +1,1247 @@
...
+| CustomerInvalidatedWritePublisherPlugin                        | Used in case if customer was invalidated or customer's password was changed and publishes customer data to storage based on customer publish event.                                                          | None          | Spryker\Zed\CustomerStorage\Communication\Plugin\Publisher\Customer |
+| EventQueueMessageProcessorPlugin                               | Used for processing invalidated customers within queue.                                                                                                                                                      | None          | Spryker\Zed\Event\Communication\Plugin\Queue                        |
+| SynchronizationStorageQueueMessageProcessorPlugin              | Registration of new queue message processor.                                                                                                                                                                 | None          | Spryker\Zed\Synchronization\Communication\Plugin\Queue              |
```
[the remark this answers is not in the document]
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/install-the-customer-account-management-feature.md:336]: CustomerStorageConfig::PUBLISH_CUSTOMER_INVALIDATED => new EventQueueMessageProcessorPlugin(),
            CustomerStorageConfig::CUSTOMER_INVALIDATED_SYNC_STORAGE_QUEUE => new SynchronizationStorageQueueMessageProcessorPlugin(),

They are registered in context of customer

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md [hunk 5]
```diff
@@ -0,0 +1,754 @@
...
+
+## 3) Set up the configuration
+
+Add the following configuration to your project:
+
+| CONFIGURATION                                           | SPECIFICATION                                                                                                                | NAMESPACE                 |
+|---------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|---------------------------|
+| TranslatorConstants::TRANSLATION_ZED_FALLBACK_LOCALES   | Fallback locales that are used if there is no translation for a selected locale.                                             | Spryker\Shared\Translator |
+| TranslatorConstants::TRANSLATION_ZED_CACHE_DIRECTORY    | An absolute path to a translation cache directory. For example, `/var/www/data/DE/cache/Zed/translation`.                       | Spryker\Shared\Translator |
+| TranslatorConstants::TRANSLATION_ZED_FILE_PATH_PATTERNS | Paths to project level translations. You can use a global pattern that specifies sets of filenames with wildcard characters. | Spryker\Shared\Translator |
```
[the remark this answers is not in the document]
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md:106]: Removed them without replacement, according to our demoshops the replacement configuration is not needed

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md [hunk 6]
```diff
@@ -0,0 +1,758 @@
...
+use Spryker\Zed\SecurityOauthUser\SecurityOauthUserConfig as SprykerSecurityOauthUserConfig;
+
+class SecurityOauthUserConfig extends SprykerSecurityOauthUserConfig
+{
+    /**
+     * Specification:
+     *  - Defines by which strategy Oauth user authentication should be.
+     *
+     * @api
+     *
```
Comment (AsonUnique) [inline _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md:166]: Usually, we don't add specification and `@api` tags to code examples

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md [hunk 7]
```diff
@@ -0,0 +1,758 @@
...
+{
+    /**
+     * @return list<\Spryker\Shared\SecurityExtension\Dependency\Plugin\SecurityPluginInterface>
+     */
+    protected function getSecurityPlugins(): array
+    {
+        return [
+            new ZedUserSessionHandlerSecurityPlugin(),
+            new ZedUserSecurityPlugin(),
+			new ZedOauthUserSecurityPlugin(),
```
Comment (AsonUnique) [inline _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md:335]: it looks like the formatting is a bit off

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md [hunk 8]
```diff
@@ -0,0 +1,758 @@
...
+
+use Spryker\Zed\User\UserDependencyProvider as SprykerUserDependencyProvider;
+use Spryker\Zed\UserLocale\Communication\Plugin\User\AssignUserLocalePreSavePlugin;
+use Spryker\Zed\UserLocale\Communication\Plugin\User\LocaleUserExpanderPlugin;
+use Spryker\Zed\UserLocaleGui\Communication\Plugin\UserLocaleFormExpanderPlugin;
+
+class UserDependencyProvider extends SprykerUserDependencyProvider
+{
+
+	/**
```
Comment (AsonUnique) [inline _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md:604]: please, check formatting in this code sample

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md [hunk 9]
```diff
@@ -0,0 +1,758 @@
...
+namespace Pyz\Zed\Console;
+
+use Spryker\Zed\Kernel\Container;
+use Spryker\Zed\Console\ConsoleDependencyProvider as SprykerConsoleDependencyProvider;
+use Spryker\Zed\Translator\Communication\Console\CleanTranslationCacheConsole;
+use Spryker\Zed\Translator\Communication\Console\GenerateTranslationCacheConsole;
+
+class ConsoleDependencyProvider extends SprykerConsoleDependencyProvider
+{
+	/**
```
Comment (AsonUnique) [inline _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md:728]: please, fix formatting in this code example

**Review by AsonUnique — DISMISSED** [review]

File: _includes/pbc/all/install-features/202407.0/install-the-customer-account-management-feature.md [hunk 10]
```diff
@@ -0,0 +1,1273 @@
...
+        return [
+            new CurrentCustomerDataRequestProcessorPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the audit logs data is expanded by the current logged in customer data.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202407.0/install-the-customer-account-management-feature.md:796]: Could you elaborate on thisz?

File: _includes/pbc/all/install-features/202407.0/install-the-customer-account-management-feature.md [hunk 11]
```diff
@@ -0,0 +1,1273 @@
...
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that, when a customer submits the login form, the URL it uses contains a locale code. For example, `/de/login_check` is the default value for the customer.
+
+{% endinfo_block %}
+
+3. You can define the password minimum and maximum length the frontend can use. They must use the same as you defined for the core feature in the previous `CustomerConfig`:
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202407.0/install-the-customer-account-management-feature.md:955]: Is this optional?

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md [hunk 12]
```diff
@@ -0,0 +1,751 @@
+{% info_block errorBox %}
+
+This feature installation guide expects the basic feature to be in place. This guide adds the following functionalities:
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md:3]: Should this list contain something related to audit logs?

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md [hunk 13]
```diff
@@ -0,0 +1,754 @@
...
+        return [
+            new CurrentUserDataRequestProcessorPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the audit logs data is expanded by the current logged in user data.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md:698]: How do I check that?

File: _includes/pbc/all/install-features/202407.0/install-the-customer-account-management-feature.md (hunk 11 above)
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/install-the-customer-account-management-feature.md:955]: yes, core module provides a default value

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md (hunk 13 above)
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md]: Fixed

File: _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md (hunk 12 above)
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md:3]: Fixed

File: _includes/pbc/all/install-features/202407.0/install-the-customer-account-management-feature.md (hunk 10 above)
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202407.0/install-the-customer-account-management-feature.md]: Fixed

**Pull request merged**